### PR TITLE
Add missing details to SAML settings

### PR DESCRIPTION
**Pull request by quique**

### CODE REVIEW

File: docs/socialaccount/providers/saml.rst [hunk 1]
```diff
@@ -72,21 +72,37 @@ via the Django admin as well:
                         # If left empty, an attempt will be done to map the attributes using
                         # built-in defaults.
                         "attribute_mapping": {
-                            "uid": "http://schemas.auth0.com/clientID",
-                            "email_verified": "http://schemas.auth0.com/email_verified",
```
Comment (pennersr) [inline docs/socialaccount/providers/saml.rst:76]: **Issue**: I think we need to keep this one...

File: docs/socialaccount/providers/saml.rst [hunk 2]
```diff
@@ -72,21 +72,37 @@ via the Django admin as well:
...
                         },
                         # The configuration of the IdP.
                         "idp": {
                             # The entity ID of the IdP is required.
-                            "entity_id": "urn:dev-123.us.auth0.com",
+                            "entity_id": "urn:example.com",
 
-                            # Then, you can either specify the IdP's metadata URL:
-                            "metadata_url": "https://dev-123.us.auth0.com/samlp/metadata/456",
+                            # Then, you can EITHER specify the IdP's metadata URL:
```
Comment (pennersr) [inline docs/socialaccount/providers/saml.rst:100]:
```suggestion
                            # Then, you can either specify the IdP's metadata URL:
```

File: docs/socialaccount/providers/saml.rst [hunk 3]
```diff
@@ -72,21 +72,37 @@ via the Django admin as well:
...
+                            "metadata_url": "https://example.com/saml2/idp/metadata.php",
 
-                            # Or, you can inline the IdP parameters here as follows:
-                            "sso_url": "https://dev-123.us.auth0.com/samlp/456",
-                            "slo_url": "https://dev-123.us.auth0.com/samlp/456",
+                            # OR, you can inline the IdP parameters here as follows:
```
Comment (pennersr) [inline docs/socialaccount/providers/saml.rst:103]:
```suggestion
                            # Or, you can inline the IdP parameters here as follows:
```

**Review by pennersr — CHANGES_REQUESTED** [review]

File: docs/socialaccount/providers/saml.rst (hunk 1 above)
Comment (quique) [reply docs/socialaccount/providers/saml.rst:76]: Ok, I committed your suggestions.

I just thought it was easy to overlook that they are alternative methods (happened to me), so I wanted to highlight it.

**Comment by pennersr** [comment]
Closed via 49f38c11

**Pull request closed without merging**